### PR TITLE
jobs: 6.1 changes and version diff

### LINKED ISSUE #4257: jobs: adjust buff tracker for mug/trick attack changes
```typescript
      trick: {
        cooldownAbility: [kAbility.TrickAttack],
        mobGainsEffect: EffectId.VulnerabilityUp,
        mobLosesEffect: EffectId.VulnerabilityUp,
```

The effect id is correct, but the ability is now mug (`0x8C8`).  On top of that, there needs to be some code changes to handle versioning, as the Chinese version is on 6.0 (and soon the Korean version too).  We have `is5x` but maybe we need to handle versioning better to handle 5.x, 6.0, and 6.1.

cc @Echoring

### PR DESCRIPTION
**Pull request by Echoring**
- use "ffxivRegion" instead of is5x.
String "intl", "cn" and "ko" can cover all conditions.
Buff tracker change "v5x" to "vCn" and "vKo", also "comboBreakers5x".
- update NIN raidbuff mug and trick attack
Fixes #4257
- update DNC proc buff
Flourishing proc keep their names, but id and icon changes. New proc Silken only change names, id and icon remains.
- update SMN Searing Light
Move to SMN himself, id and icon changed but effect id not.
- update WHM lily and wind dot
Get lily per 20s now. Wind dot 30s now.
- update AST combust1 to 30s
- PLD spell no longer break combos, and add combo timer.
So PLD finally have insertable skill, and combo timer also become useable.
- DRK update blood weapon
Become stack, duration no longer necessary.

### CODE REVIEW

**Comment by quisquous** [comment]
This is a lot of changes at once.  I won't ask you to break this up, but in the future I'd definitely prefer having refactoring (intl/cn/ko), bugfixes (mug), other job changes in separate PRs.  It's harder to review and if one of these changes has a bug, it makes it harder to track down which part of the PR caused the issue.  If there's separate PRs, it's much easier to narrow it down and understand what the changes were.

**Comment by Echoring** [comment]
Sorry for such a large pr, but I'm not sure how to deal with the situation that some fixing rely on refactor.
If they are given in seperate prs, won't them meet conflict? Or if one of them reverted, will other get bugs?

**Comment by quisquous** [comment]
Yeah, I think it's not a big deal.  I think it's more that smaller PRs are generally better.  It's bugs, but it's also having separate changes makes them more obvious in the changelog and also if I have questions on one part (e.g. drk) it doesn't hold up all of the other parts from landing too.  (Ack, somehow those review comments didn't get published...*hits button*)

I think I could have imagined this being a: "refactor to add version differences, but no other changes" and then landing separately things that used that on top, which you've already split into commits.

File: ui/jobs/components/drk.ts [hunk 1]
```diff
@@ -86,24 +86,28 @@ export class DRKComponent extends BaseComponent {
...
+        } else {
+          this.bloodWeapon.duration = 10;
+          this.bloodWeapon.threshold = 10;
           this.bloodWeapon.fg = computeBackgroundColorFrom(
             this.bloodWeapon,
-            'drk-color-bloodweapon',
+            'drk-color-bloodweapon.active',
           );
-        }, 10000);
+          this.tid1 = window.setTimeout(() => {
```
[the remark this answers is not in the document]
Comment (Echoring) [reply ui/jobs/components/drk.ts:98]: Github not show this change well. The change is, if cn/ko, nothing change, and if intl, no longer trace duration.
Bloodweapon is almostly sure to be cosumed before expire, and if you can't, nothing can prevent it fall off.
So it no longer worth tracing. By the way, I prepared to remove one of Bloodweapon/Delirium to handle Salted Earth.

[the remark this answers is not in the document]
Comment (quisquous) [reply ui/jobs/components/drk.ts:98]: Oh, thank you for the patient explanation.  Sorry for misunderstanding this code change.  :+1:

**Pull request merged**